### PR TITLE
Цветков М.С. 381803-2. ЛР1.

### PR DESCRIPTION
**Pull request by TsvetkovMaximPmi3**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [x] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [x] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by TsvetkovMaximPmi3** [comment]
#146

**Comment by TsvetkovMaximPmi3** [comment]
@Iliamish 
@dmitrybezro 
@alxndrvna

**Comment by streltsova-yana** [comment]
Необходимо обновить локально main и сделать с ним rebase:
`git checkout main`
`git pull remote main`
`git rebase main my_branch_name`
`git push -f origin my_branch_name`

**Review by alxndrvna — APPROVED** [review]

File: modules/complex-number/test/test_tsvetkov_maksim_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+#include "include/complex_number.h"
+
+TEST(Tsvetkov_Maksim_ComplexNumberTest, Pow_of_imaginary_unit) {
+    // Arrange
+    ComplexNumber z(0, 1.0);
+    bool res;
+
+    // Act
+    res = z == z*z*z*z*z*z*z*z*z;
```
Comment (Iliamish) [inline modules/complex-number/test/test_tsvetkov_maksim_complex_number.cpp:14]: If you set ComplexNumber z(0, 1.0); even if `*` returns `z` this test will pass. I think setting number to (0,2) and after n multiplications compare it to (0,2)^n is more correct

Comment (dkurt) [reply modules/complex-number/test/test_tsvetkov_maksim_complex_number.cpp:14]: And please use `for` loop in the name of the machine god.

**Pull request merged**